adapter/noop2/noop.go

```diff
@@ -82,37 +84,41 @@ func (builder) ConfigureQuotaHandler(map[string]*quota.Type) error {
 
 ////////////////// Runtime Methods //////////////////////////
 
-var cacheInfo = adapter.CacheabilityInfo{
+var checkResult = adapter.CheckResult{
+	Status:        rpc.Status{Code: int32(rpc.OK)},
 	ValidDuration: 1000000000 * time.Second,
 	ValidUseCount: 1000000000,
 }
+var reportResult = adapter.ReportResult{
```

Comment:
If this was proto, a wrapper message is a requirement and we argued that a ReportResult is future proofing. But since this is just go struct, the only rationale is consistency with other methods ? @mandarjog your thoughts ?
Plus other methods are not returing rpc.status to denote permission errors etc.. I think repor handler might also want to return status right ? if we don't make this ReportResult, should we at least make this rpc.Status ?